src/Components/Landing.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from '../css/Landing.module.css';
+
+export const Landing = (): React.ReactElement => (
+  <div className={styles.Landing}>
+    <NavBar />
+    <div>
+      <h1 className={styles.Title}>
+        #SpreadThe
+        <span className={styles.Love}>Love</span>
+      </h1>
+      <p className={styles.SubText}>
+        Week of June 20, 2020
+      </p>
+      <p className={styles.SubText}>
+        Happy mid-June. Whether you're stuck at home or your town is reopening, here are some
```

Comment:
linter error--apostrophe needs to be escaped

Reply:
Fixed this